kyaml/kio/kio.go

```diff
@@ -6,6 +6,9 @@
 package kio
 
 import (
+	"path/filepath"
+
+	"sigs.k8s.io/kustomize/api/filesys"
```

Comment:
I have a question for @KnVerey and @monopole.

It seems that one way or another, for the functionality desired here, kyaml will have to depend on api.

But based on the releasing notes, [api already depends on kyaml](https://github.com/kubernetes-sigs/kustomize/tree/master/releasing). Does that change anything about releasing? Is this dependency okay?

Reply:
I know that there was a plan to eventually move `api/filesys` to `kyaml/filesys`, this PR does not make that process easier or harder so it could be done next.

Reply:
Is there a tracking issue for the move? If we're going to do that it should probably happen before this PR

Reply:
There's no issue, but I did contribute this PR: https://github.com/kubernetes-sigs/kustomize/pull/3997